test/parallel/test-assert.js

```diff
@@ -378,7 +378,7 @@ try {
 assert.strictEqual(true, threw,
                    'a.doesNotThrow is not catching type matching errors');
 
-assert.throws(function() { assert.ifError(new Error('test error')); });
+assert.throws(function() { assert.ifError(new Error('test error')), /^Error: test error/});
```

Comment:
I think the regular expression should come after the `}`. Also, please add a `$` to the end of the regex.